src/esbmc/bmc.cpp

```diff
@@ -1443,3 +1481,42 @@ smt_convt::resultt bmct::multi_property_check(
 
   return final_result;
 }
+
+void bmct::report_simple_summary(const SimpleSummary &summary) const
+{
+  if (options.get_bool_option("result-only"))
+    return;
+
+  // ANSI color codes
+  const std::string GREEN = "\033[32m";
+  const std::string RED = "\033[31m";
+  const std::string RESET = "\033[0m";
+
+  // Build the properties summary string with colors
+  std::ostringstream properties_oss;
+  properties_oss << "Properties: " << summary.total_properties << " verified";
+
+  if (summary.passed_properties > 0)
+    properties_oss << " " << GREEN << "✓ " << summary.passed_properties
+                   << " passed" << RESET;
+
+  if (summary.failed_properties > 0)
+    properties_oss << ", " << RED << "✗ " << summary.failed_properties
```

Comment:
Like for example, we can use `log_error` here to make the output in red.

Or we can keep this implementation, but only enable it when enabling `--color`

Reply:
Thanks, @ChenfengWei0. Could you please submit a small PR with the --color suggestion?